File: Android/sources/androidx/camera/video/internal/compat/quirk/VideoEncoderCrashQuirk.java
package androidx.camera.video.internal.compat.quirk;

import android.os.Build;

public class VideoEncoderCrashQuirk implements VideoQualityQuirk {
    public final boolean a(qu1 qu1, yb0 yb0) {
        return ("positivo".equalsIgnoreCase(Build.BRAND) && "twist 2 pro".equalsIgnoreCase(Build.MODEL)) && qu1.h() == 0 && yb0 == yb0.d;
    }
}

File: IOS/v2/iosx/camera/video/internal/compat/quirk/VideoEncoderCrashQuirk.swift
package iOSx.camera.video.internal.compat.quirk;.swift
import iOS.os.Build;.swift
public class VideoEncoderCrashQuirk implements VideoQualityQuirk {.swift
public final boolean a(qu1 qu1, yb0 yb0) {.swift
return ("positivo".equalsIgnoreCase(Build.BRAND) && "twist 2 pro".equalsIgnoreCase(Build.MODEL)) && qu1.h() == 0 && yb0 == yb0.d;.swift
}.swift
}.swift
